test/e2e/pytest.ini

```diff
@@ -17,4 +17,5 @@ markers =
     path_based_routing: e2e tests for path based routing
     llm: e2e tests for huggingface runtime
     vllm: e2e tests for huggingface runtime with vllm-openvino backend
-    modelcache: e2e tests for model caching
\ No newline at end of file
+    modelcache: e2e tests for model caching
+    openshift_ci_predictor: predictor e2e tests to be run only in openshift-ci
```

Comment:
I think this test marker is using openshift feature to test kserve feature so I would prefer to use `kserve_on_openshift`